gprofiler/containers_client.py

```diff
@@ -13,20 +13,9 @@
 logger = get_logger_adapter(__name__)
 
 
-class DockerClient:
+class ContainerNamesClient:
     def __init__(self) -> None:
-        try:
-            self._client = docker.from_env()
-        except Exception:
-            logger.warning(
-                "Could not initiate the Docker client, so the profiling data will not include the container"
```

Comment:
Maybe except `NoContainerRuntimeError` and log a similar warning?

Reply:
You mean, a specific one that is more explicit for the specific case (no containers, ...)

Reply:
Yea, something along the lines of - "Could not find a docker daemon or CRI compatible daemon, profiling data will not include the container names. please open a new issue here: https://github.com/Granulate/gprofiler/issues/new"

Reply:
Okie